cmake/Version.cmake

```diff
@@ -8,16 +8,20 @@
 
 string(TIMESTAMP current_year "%Y")
 set(LY_VERSION_COPYRIGHT_YEAR ${current_year} CACHE STRING "Open 3D Engine's copyright year")
-set(LY_VERSION_STRING "0.0.0.0" CACHE STRING "Open 3D Engine's version")
+set(LY_VERSION_STRING "0.0.0" CACHE STRING "Open 3D Engine's version")
+set(LY_DISPLAY_VERSION_STRING "00.00" CACHE STRING "Open 3D Engine's display version")
```

Comment:
Can we name this variable `O3DE_DISPLAY_VERSION_STRING`?

It is strange that the environment variable is `O3DE_DISPLAY_VERSION`, while the cache variable is `LY_DISPLAY_VERSION_STRING`.

I think they should both start with `O3DE_`